course/admin.py

```diff
@@ -6,8 +6,32 @@
 from course.models.models import Question, VariableQuestion, MultipleChoiceQuestion, Submission, QuestionCategory, \
     CheckboxQuestion, JavaSubmission, JavaQuestion, TokenValue, MultipleChoiceSubmission, UserQuestionJunction
 from course.models.parsons_question import ParsonsQuestion, ParsonsSubmission
+import copy  # Python native copy functionality
 
 
+def duplicate_question(modeladmin, request, queryset):
```

Comment:
Having an admin action is nice. But what we really need is to have the ability to duplicate questions in the course.
Our TAs/Instructors do not have access to the admin page. The admin page is restricted to the developers.